web/src/lib/components/asset-viewer/photo-viewer.svelte

```diff
@@ -9,13 +9,15 @@
 
   export let asset: AssetResponseDto;
   export let publicSharedKey = '';
-  let imgElement: HTMLDivElement;
+  export let imgElement: HTMLDivElement;
 
   let assetData: string;
 
   let copyImageToClipboard: (src: string) => Promise<Blob>;
   let canCopyImagesToClipboard: () => boolean;
 
+  $: console.log(imgElement);
+
   onMount(async () => {
     // Import hack :( see https://github.com/vadimkorr/svelte-carousel/issues/27#issuecomment-851022295
```

Comment:
This is funny and can probably be removed given the web uses ESM.

Reply:
removed ;)